website/docs/getting-started/setup/installation-guides/docker/README.mdx

```diff
@@ -81,9 +81,12 @@ The Appsmith server should be up and running soon and can be accessed at [http:/
 
 ### Updating Appsmith
 :::caution
-   It's recommended to backup the Appsmith instance before performing an update. For more information, see [How to create a backup](https://docs.appsmith.com/getting-started/setup/instance-management/appsmithctl#backup-appsmith-instance).
+   * It's recommended to backup the Appsmith instance before performing an update. For more information, see [How to create a backup](https://docs.appsmith.com/getting-started/setup/instance-management/appsmithctl#backup-appsmith-instance).
```

Comment:
Replace with below

-  [Create a backup](https://docs.appsmith.com/getting-started/setup/instance-management/appsmithctl#backup-appsmith-instance) of the Appsmith instance before performing a manual update.
- If your version is less than **v1.9.2**, [upgrade to the checkpoint version](https://docs.appsmith.com/getting-started/setup/instance-management#checkpoint-upgrade) v1.9.2 before upgrading to the latest version.